OrbitGrpcProtos/capture.proto

```diff
@@ -122,6 +124,32 @@ message ThreadName {
   uint64 timestamp_ns = 4;
 }
 
+message ThreadStateSlice {
+  int32 pid = 1;
+  int32 tid = 2;
```

Comment:
The pid is not set as we don't have it, but I want to have the field right away as it's only because of technical reason and we'll probably set it in the future. Let's remember that these are protos v3 and all fields are optional.

Reply:
Added comment.